src/state_machine.cpp

```diff
@@ -651,6 +663,26 @@ void Base::handleReadCard() {
   }
 }
 
+bool Base::checkRawForSortcutAndShutdown(commandRaw cmd_raw) {
+  uint8_t shortCut = 0xff;
+  switch(cmd_raw) {
+  case commandRaw::updownLong: shortCut = 0      ; break;
+  case commandRaw::upLong    : shortCut = 1      ; break;
+  case commandRaw::downLong  : shortCut = 2      ; break;
+  case commandRaw::start     : shortCut = 3      ; break;
+  case commandRaw::pauseLong : tonuino.shutdown(); break;
```

Comment:
Maybe play a jingle when shutting down manually?

Reply:
Added in shutdown function. Maybe later replaced by "bye bye".

Reply:
Great! Should that also be played when the standby timer triggers the shutdown? I think it shouldn't.